invenio_app_rdm/records_ui/views/deposits.py

```diff
@@ -483,7 +488,12 @@ def deposit_edit(pid_value, draft=None, draft_files=None, files_locked=True):
     can_edit_draft = service.check_permission(
         g.identity, "update_draft", record=draft._record
     )
+    can_preview_draft = service.check_permission(
+        g.identity, "preview", record=draft._record
+    )
     if not can_edit_draft:
+        if can_preview_draft:
+            return redirect(f"/records/{pid_value}?preview=1")
```

Comment:
it seems that there is no `preview` link

Reply:
I guess you mean in the links of the record no? Could we add it?

Reply:
> I guess you mean in the links of the record no? Could we add it?

plus one to this